### ISSUE TITLE
Incomplete right bundle branch block

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7585]
**Preferred term label:**
Incomplete right bundle branch block

**Synonyms**

**Definition (free text, please give PubMed ID)**

Right bundle branch block (RBBB) represents an abnormality in the intraventricular electrical conduction system of the human heart, generating enlarged QRS and modified vectors on the electrocardiogram (ECG). Incomplete RBBB (IRBBB) is defined by (i) QRS duration between 110 and 120 ms in adults; (ii) rsr, rsR, or rSR in leads V1 or V2. The R or r deflection is usually wider than the initial R wave. In a minority of patients, a wide and often notched R wave pattern may be seen in leads V1 and/or V2; (iii) S wave of greater duration than R wave or greater than 40 ms in leads I and V6 in adults; and (iv) Normal R peak time in leads V5 and V6 but greater than 50 ms in lead V1. The first 3 criteria should be present to make the diagnosis. When a pure dominant R wave with or without a notch is present in V1, criterion 4 should be satisfied
Comment
Right bundle branch block  appears with a prevalence of 3% to 7% in the general population, more frequently in men than in women. .

PMID:34100724

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:601494 
PMID: 11684629  2/20

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000313
- label: Incomplete right bundle branch block
- parents: [Bundle branch block (HP:0011710)](https://hpo.jax.org/app/browse/term/HP:0011710)
- pmid: PMID:34100724
- def: Right bundle branch block (RBBB) represents an abnormality in the intraventricular electrical conduction system of the human heart, generating enlarged QRS and modified vectors on the electrocardiogram (ECG). Incomplete RBBB (IRBBB) is defined by (i) QRS duration between 110 and 120 ms in adults; (ii) rsr, rsR, or rSR in leads V1 or V2. The R or r deflection is usually wider than the initial R wave. In a minority of patients, a wide and often notched R wave pattern may be seen in leads V1 and/or V2; (iii) S wave of greater duration than R wave or greater than 40 ms in leads I and V6 in adults; and (iv) Normal R peak time in leads V5 and V6 but greater than 50 ms in lead V1. The first 3 criteria should be present to make the diagnosis. When a pure dominant R wave with or without a notch is present in V1, criterion 4 should be satisfied.
- comment: Right bundle branch block appears with a prevalence of 3% to 7% in the general population, more frequently in men than in women.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7585
- synonyms:
